### DISCUSSION TITLE
nice project! i want to help!

### DISCUSSION THREAD

**Discussion by SilkePilon** [whatotter/pwnhyve#14] [General]
i want to help with the front end!

**Comment by whatotter** [comment]
thank you! i already have a friend who will help when the time comes (should be in a few days or so, after the next update) - you guys can hopefully work together n just make a simplistic frontend
we dont have a place to really chat rn, but i'll probably make something like a discord server
ill gladly let you know

**Reply by loguecow** [reply]
Any server yet?

**Reply by whatotter** [reply]
https://discord.gg/HdPKgHQj

**Comment by SilkePilon** [comment]
👍

**Comment by EpicTia93** [comment]
> SilkePilon

Hey man let me know too if you make a discord server! Maybe I can do smth to help too.

**Comment by sputnikOS** [comment]
Ahoy. I'm looking for some open source projects to contribute to and I would love to help as well. I've been building Angular UIs since version 1. If there's anything I can do to help or collaborate with SilkePilon, please let me know! Cheers 🍻

**Comment by whatotter** [comment]
converted to discussion just in case other people want to join

**Comment by Wesley5n1p35** [comment]
Discord?

**Comment by dj1ch** [comment]
yo, another small developer like you(@whatotter) working on embedded systems, microcontrollers, sbc's, etc. just submitted a pr if you want to merge. lmk if you want more help on this project, i got a fair share of programming experience. i also just so happen to be in the pwnagotchi-unnofficial organization. if you have any questions about the pwnagotchi feel free to ask me or the rest of the team on the discord server

**Comment by caseybarajas** [comment]
I would like to help also.

**Comment by whatotter** [comment]
okay, discord server has been made:

https://discord.gg/HdPKgHQj

link is set to never expire :)
